### ISSUE TITLE
TypeError: Cannot read property 'children' of undefined

### ISSUE THREAD

**Issue by Mrtenz** [dream-bit-de/gatsby-plugin-better-page-tree#1]
Simply installing the plugin and adding it to `gatsby-config` results in the following error:

```
 ERROR #11321  PLUGIN

"@dream-bit-de/gatsby-plugin-better-page-tree" threw an error while running the onCreateNode lifecycle:

Cannot read property 'children' of undefined

  46 |     PageTreeMap[currentPage === '' ? 'home' : currentPage] = pageTreeNode;
  47 |   } else {
> 48 |     createParentChildLink({
     |     ^
  49 |       parent: PageTreeMap[parentPage === '' ? 'home' : parentPage],
  50 |       child: pageTreeNode
  51 |     });

File: node_modules/@dream-bit-de/gatsby-plugin-better-page-tree/gatsby-node.js:48:5

  TypeError: Cannot read property 'children' of undefined
```

**Comment by imshuffling** [comment]
Getting the same error being thrown, any joy with a solution?

**Comment by Mrtenz** [comment]
I haven't found a solution yet unfortunately, and ended up with a custom solution.

**Comment by Megajin** [comment]
Which version did you guys use?
Unfortunately the notifications were turned off. I'm sorry for that.

This was a known bug when used together with transform plugins. However the newest version should not behave like that.

**Comment by Megajin** [comment]
@imshuffling and @Mrtenz , I will close this Issue within 5 days. I still believe that your problems are solved with the latest update. I'd like to hear from you guys, if you were able to overcome this problem with the new version. As I said, this took so long because the notification for this repository was turned off, that was completely my fault. Again, I'm deeply sorry for that.

**Comment by Mrtenz** [comment]
Thanks for the reply. I'm currently not using this plugin so I have no way to test it, but maybe imshuffling can give some insights. Otherwise, feel free to close the issue.

**Comment by Megajin** [comment]
Strangely I did run into this Issue just a moment ago. However everything was working just fine a few minutes ago. After a short research I did find out that this Issue is related to: https://github.com/gatsbyjs/gatsby/issues/10506

I was able to fix this problem by simply deleting the `.cache` folder.

**Comment by Megajin** [comment]
Another Tip for everyone using the older version:

This error only occurs if the child pages are generated before the parent page.
However this is fixed in v1.6.0.
